src/index.html

```diff
@@ -0,0 +1,452 @@
+          High security to protect from piracy
+        </p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy.
+        </p>
+      </div>
+      <div class="border-l-[3px] border-[#CDC7D2] pl-[30px]">
+        <img src="./img/3.png" class="w-[66.44px] mb-[20px]" />
+        <p class="text-[32px] font-[700] leading-[120%] mt-[10px] mb-[20px]">
+          High security to protect from piracy
+        </p>
+        <p class="font-[400] text-[16px] opacity-[0.87] mt-[10px]">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy.
+        </p>
+      </div>
+    </div>
+    <p
+      class="text-[56px] font-[600] leading-[110%] mb-[20px] text-[#1D2130] text-center"
+    >
+      The stunning results our <br />
+      customers have experienced
+    </p>
+    <section
+      class="grid grid-cols-3 text-center mr-[5%] ml-[5%] mt-[80px] gap-[20px]"
+    >
+      <div
+        class="border-[2px] border-bg-rgba-3 p-[50px] rounded-[24px] pl-[40px] pr-[40px]"
+      >
+        <p class="font-[600] text-[18px] leading-[150%] mb-[20px]">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </p>
+        <img src="./img/people.png" class="w-[56px] m-auto" />
+        <p class="font-[500] text-[20px] leading-[150%]">Ron Wood</p>
+        <p class="font-[400] text-[#1D2130] text-[12px] leading-[18px]">CEO</p>
+      </div>
+      <div
+        class="border-[2px] border-bg-rgba-3 p-[50px] rounded-[24px] pl-[40px] pr-[40px]"
+      >
+        <p class="font-[600] text-[18px] leading-[150%] mb-[20px]">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </p>
+        <img src="./img/people.png" class="w-[56px] m-auto" />
+        <p class="font-[500] text-[20px] leading-[150%]">Ron Wood</p>
+        <p class="font-[400] text-[#1D2130] text-[12px] leading-[18px]">CEO</p>
+      </div>
+      <div
+        class="border-[2px] border-bg-rgba-3 p-[50px] rounded-[24px] pl-[40px] pr-[40px]"
+      >
+        <p class="font-[600] text-[18px] leading-[150%] mb-[20px]">
+          Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,
+          sed diam voluptua.
+        </p>
+        <img src="./img/people.png" class="w-[56px] m-auto" />
+        <p class="font-[500] text-[20px] leading-[150%]">Ron Wood</p>
+        <p class="font-[400] text-[#1D2130] text-[12px] leading-[18px]">CEO</p>
+      </div>
+    </section>
+    <footer class="mt-[130px] text-center relative">
+      <div class="footer__background"></div>
```

Comment:
Where is the block of this element? you should have "footer" block in order to use "footer__background" element